yolox/evaluators/coco_evaluator.py

```diff
@@ -280,10 +280,14 @@ def evaluate_prediction(self, data_dict, statistics):
             with contextlib.redirect_stdout(redirect_string):
                 cocoEval.summarize()
             info += redirect_string.getvalue()
+            cat_ids = list(cocoGt.cats.keys())
+            cat_names = [cocoGt.cats[catId]['name'] for catId in sorted(cat_ids)]
             if self.per_class_AP:
-                info += "per class AP:\n" + per_class_AP_table(cocoEval) + "\n"
+                per_class_AP_table_str = per_class_AP_table(cocoEval, class_names=cat_names)
```

Comment:
Due to line lenght limits `per_class_AP_table_str ` var was added. Potentially it can be futher used for returning per_class AP/AR for fancy logging.